src/services/directory.ts

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import {
+    backendFetch,
+    backendFetchJson,
+    getRequestParamFromList,
+} from './utils';
+import { UUID } from 'crypto';
+
+const PREFIX_DIRECTORY_SERVER_QUERIES =
+    import.meta.env.VITE_API_GATEWAY + '/directory';
+
+function getPathUrl(elementUuid: UUID) {
+    return `${PREFIX_DIRECTORY_SERVER_QUERIES}/v1/elements/${encodeURIComponent(
+        elementUuid
+    )}/path`;
+}
+
+export function fetchRootFolders(types: string[]) {
```

Comment:
```suggestion
export function fetchRootFolders(types: string[]): Promise<???> {
```
we don't want a `Promise<any>`.